cypress/support/fragments/data-export/selectJobProfile.js

```diff
@@ -1,10 +1,20 @@
-import { Pane, Button, MultiColumnList, PaneHeader, TextFieldIcon } from '../../../../interactors';
+import {
+  Pane,
+  Button,
+  MultiColumnList,
+  PaneHeader,
+  TextFieldIcon,
+  MultiColumnListCell,
+} from '../../../../interactors';
 
 const searchResults = MultiColumnList({ id: 'search-results-list' });
 const jobProfilesPaneHeader = PaneHeader({ id: 'paneHeaderpane-results-subtitle' });
 const searchField = '[class^=formControl]';
 const searchIcon = TextFieldIcon();
-const searchButton = Button('Search', { disabled: true });
+const jobProfilescSearchId = '#input-search-field';
+const xIconSelector = '[class^="endControls"]>div>div';
```

Comment:
Maybe there's another way to use this selector? I'm confused about div's.

Reply:
I haven't found the way to do so
Open to any suggestions!